controllers/translateController.js

```diff
@@ -53,11 +54,14 @@ const translateController = async ({
     cliArgEngine: options.engine,
   });
 
+  const config = await getConfig();
+
   for await (const language of languages) {
     const filesToEdit = await validateAndPromptUserJSONFiles(
       basePath,
       language.files,
       nameOfTranslation,
+      config,
```

Comment:
We already import the config in this part: 

https://github.com/heliomar-pena/i18n-populator/blob/d7afb852edac76e6c03c5f74591a0f5466c50a01/controllers/translateController.js#L41-L45

So you don't need to import it here ^^